app/src/test/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryUiRendererTest.kt

```diff
@@ -85,6 +86,7 @@ class BloodSugarEntryUiRendererTest {
     bloodSugarEntryUiRenderer.render(bloodSugarEntryModel)
 
     // then
+    verify(ui).showRemoveButton()
```

Comment:
Should we add separate tests for this?

`When the sheet is opened for editing a blood sugar measurement, the remove button must be shown`, and `When the sheet is opened for recording a new blood sugar measurement, the remove button must not be shown`

Reply:
Discussed offline: the tests are already present and were written earlier.